lib/iris/tests/experimental/test_animate.py

```diff
@@ -40,6 +40,7 @@
 @tests.skip_plot
 class IntegrationTest(tests.GraphicsTest):
     def setUp(self):
+        super(IntegrationTest, self).setUp()
```

Comment:
> I'd argue that this is a non-issue

Agreed. I was sure there must be a pattern to it, but I couldn't work out what the pattern was! Good ol' inheritance :wink: